src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -105,35 +120,61 @@ public string RMProfileModule
         {
             get
             {
-                return Path.Combine(this.PackageDirectory,
+#if !NETSTANDARD
+               return Path.Combine(this.PackageDirectory,
                                     @"ResourceManager\AzureResourceManager\AzureRM.Profile\AzureRM.Profile.psd1");
+#else
+                return Path.Combine(this.PackageDirectory,
+                                    "ResourceManager","AzureResourceManager", "AzureRM.Profile.Netcore", "AzureRM.Profile.Netcore.psd1");
+#endif
```

Comment:
Just thinking in general... we could make some provider in our common stuff that does path resolution. So, other modules don't need to resolve differences between .Netcore/Stack/Package/etc. paths to access the same objects.